Applications/WebPortal/WebApp/library/source/pages/account_page/compliance_page/compliance_page.tsx

```diff
@@ -13,24 +13,28 @@ interface Properties {
   currencyDatabase?: Nexus.CurrencyDatabase;
 
   /** The list of compliance rules to display and edit. */
-  complianceList: Nexus.ComplianceRuleEntry[];
+  entries: Nexus.ComplianceRuleEntry[];
 
-  /** A list of rule schemas. Used in adding new rules. */
-  ruleSchemas: Nexus.ComplianceRuleSchema[];
+  /** The list of rule schemas. Used in adding new rules. */
+  schemas: Nexus.ComplianceRuleSchema[];
+
+  /** The callback for adding the rule.*/
+  onRuleAdd?: (newSchema: Nexus.ComplianceRuleSchema) => void;
```

Comment:
Be consistent about the tense you use, either consistently use the past tense (as you did for ```onRuleChanged``` or use the present tense (as you did for ```onRuleAdd```). This consistency needs to be applied to all callbacks throughout the entire codebase.

Reply:
Do you have a preference? I think present tense makes more sense.

Reply:
Check what convention is used for other components and go for it.